### PR TITLE
Release 0.18.6

### PR DESCRIPTION
**Pull request by ankatiyar**
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
> [`kedro-plugins` repository](https://github.com/kedro-org/kedro-plugins). Any changes to the dataset implementations
> should be done by opening a pull request in that repository.
## Description

Release Kedro 0.18.6 
## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -8,15 +8,33 @@
...
+## Migration guide from Kedro 0.18.5 to 0.18.6
+A regression introduced in Kedro version `0.18.5` caused the `Kedro-Viz` console to fail to show experiment tracking correctly. If you experienced this issue, you will need to:
+* upgrade to Kedro version `0.18.6`
+* delete any erroneous session entries created with Kedro 0.18.5 from your session_store.db stored at `<project-path>/data/session_store.db`.
+
+Thanks to Kedroids tomohiko kato, [tsanikgr](https://github.com/tsanikgr) and [maddataanalyst](https://github.com/maddataanalyst) for very detailed reports about the bug.
+
+## Upcoming deprecations for Kedro 0.19.0
+* `project_version` will be deprecated in `pyproject.toml` please use `kedro_init_version` instead.
+* Deprecated `kedro run` flags `--node`, `--tag`, and `--load-version` in favour of `--nodes`, `--tags`, and `--load-versions`.
```
Comment (astrojuanlu) [inline RELEASE.md:35]: These are in 0.18.5 too, is the repetition intentional?

Comment (ankatiyar) [reply RELEASE.md:35]: I think @stichbury might have added this but I also don't think we need this - should I remove it?

File: docs/source/tutorial/tutorial_template.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Don't forget to check the [tutorial FAQ](spaceflights_tutorial_faqs.md) if you r
 If you have not yet set up Kedro, do so by [following the guidelines to install Kedro](../get_started/install.md).
 
 ```{important}
-We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.18.5).
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.18.6).
```
Comment (AhdraMeraliQB) [inline docs/source/tutorial/tutorial_template.md:20]: I think it's fine as this release was very quick (and fixes a bug in 0.18.5), but in future it probably defeats the purpose to update this every time we do the release 🤔

File: RELEASE.md (hunk 1 above)
Comment (AhdraMeraliQB) [reply RELEASE.md]: I'm of the opposite mind - I don't think we should remove it until 0.19.0 is out as the to-be-deprecated features are still there and there is still room for new users to get exposed to those features pre-0.19.0.

Comment (ankatiyar) [reply RELEASE.md]: We haven't been copying over all the deprecated things from the previous 0.18.x releases so far though

**Review by rashidakanchwala — APPROVED** [review]
thanks :)

**Review by merelcht — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**